tests/integration/suite/daprd/workflow/noawait.go

```diff
@@ -50,7 +50,7 @@ func (n *noawait) Run(t *testing.T, ctx context.Context) {
 
 	var barCalled atomic.Int64
 	n.workflow.Registry().AddOrchestratorN("noawait", func(ctx *task.OrchestrationContext) (any, error) {
-		for range 50 {
+		for range 30 {
```

Comment:
What? We have a test like this? Calling activities and not awaiting them? This shouldnt event be supported

Reply:
Indeed, this is somewhat undefined behaviour... this test will at least ensure behaviour doesn't change until we have some kind of mechanism to do something different